packages/next-auth/src/providers/boxyhq-saml.ts

```diff
@@ -0,0 +1,37 @@
+import type { OAuthConfig, OAuthUserConfig } from "."
+
+export interface SAMLJacksonProfile {
+  id: string;
+  email?: string;
```

Comment:
You mentioned `email` is guaranteed at https://github.com/nextauthjs/next-auth/pull/3782#discussion_r799064357
```suggestion
  email: string
```